runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowStreamingPipelineOptions.java

```diff
@@ -211,6 +211,14 @@ public interface DataflowStreamingPipelineOptions extends PipelineOptions {
 
   void setWindmillServiceStreamMaxBackoffMillis(int value);
 
+  @Description(
+      "If true, Dataflow streaming pipeline will be running in direct path mode."
+          + " VMs must have IPv6 enabled for this to work.")
+  @Default.InstanceFactory(IsWindmillServiceDirectPathEnabled.class)
```

Comment:
We don't need the factory `IsWindmillServiceDirectPathEnabled`.

Reply:
ack ok

Reply:
changed